sickbeard/tvcache.py

```diff
@@ -178,7 +178,7 @@ def updateCache(self):
                 cl = []
                 index = 0
                 for index, item in enumerate(data['entries'] or []):
-                    if recent_results and item.get('link').strip() in [cached_item['link'].strip() for cached_item in recent_results]:
+                    if recent_results and item['link'] in {cache_item['link'] for cache_item in recent_results}:
```

Comment:
no need to check if recent_results.  Recent results is a list and if its empty `{cache_item['link'] for cache_item in recent_results}` would be an empty list anyway so the test would already eval false.